docs/source/cpp/parquet.rst

```diff
@@ -450,38 +450,46 @@ Logical types
++-------------------+-----------------------------+----------------------------+----------+
+| INT               | INT64                       | Int64 / UInt64             |          |
++-------------------+-----------------------------+----------------------------+----------+
+| DECIMAL           | INT32 / INT64 / BYTE_ARRAY  | Decimal128 / Decimal256    | \(2)     |
+|                   | / FIXED_LENGTH_BYTE_ARRAY   |                            |          |
++-------------------+-----------------------------+----------------------------+----------+
+| DATE              | INT32                       | Date32                     | \(3)     |
++-------------------+-----------------------------+----------------------------+----------+
+| TIME              | INT32                       | Time32 (milliseconds)      |          |
++-------------------+-----------------------------+----------------------------+----------+
+| TIME              | INT64                       | Time64 (micro- or          |          |
+|                   |                             | nanoseconds)               |          |
++-------------------+-----------------------------+----------------------------+----------+
+| TIMESTAMP         | INT64                       | Timestamp (milli-, micro-  |          |
+|                   |                             | or nanoseconds)            |          |
++-------------------+-----------------------------+----------------------------+----------+
+| STRING            | BYTE_ARRAY                  | String / LargeString /     |          |
+|                   |                             | StringView                 |          |
++-------------------+-----------------------------+----------------------------+----------+
+| LIST              | Any                         | List                       | \(4)     |
++-------------------+-----------------------------+----------------------------+----------+
+| MAP               | Any                         | Map                        | \(5)     |
++-------------------+-----------------------------+----------------------------+----------+
+| FLOAT16           | FIXED_LENGTH_BYTE_ARRAY     | HalfFloat                  |          |
++-------------------+-----------------------------+----------------------------+----------+
+| UUID              | FIXED_LENGTH_BYTE_ARRAY     | Extension (arrow.uuid)     | \(6)     |
++-------------------+-----------------------------+----------------------------+----------+
+| JSON              | BYTE_ARRAY                  | Extension (arrow.json)     | \(6)     |
++-------------------+-----------------------------+----------------------------+----------+
+| GEOMETRY          | BYTE_ARRAY                  | Extension (geoarrow.wkb)   | \(6) \(7)|
++-------------------+-----------------------------+----------------------------+----------+
+| GEOGRAPHY         | BYTE_ARRAY                  | Extension (geoarrow.wkb)   | \(6) \(7)|
```

Comment:
This will probably look better by using literal quotes (you'll need to reformat the table though).
```suggestion
| UUID              | FIXED_LENGTH_BYTE_ARRAY     | Extension (``arrow.uuid``) | \(6)     |
+-------------------+-----------------------------+----------------------------+----------+
| JSON              | BYTE_ARRAY                  | Extension (``arrow.json``) | \(6)     |
+-------------------+-----------------------------+----------------------------+----------+
| GEOMETRY          | BYTE_ARRAY                  | Extension (``geoarrow.wkb``) | \(6) \(7)|
+-------------------+-----------------------------+----------------------------+----------+
| GEOGRAPHY         | BYTE_ARRAY                  | Extension (``geoarrow.wkb``) | \(6) \(7)|
```